Update getting_started_introduction.md (#6037)
qmk/qmk_firmware: docs/getting_started_introduction.md

### Keyboard Project Structure

> Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it you'll find the following structure:

Reviewer: Was gonna let this one go, but nope, it still bothers me (not your fault; it's been like this for a while).

```suggestion
Within the folder `keyboards`, its subfolder `handwired` and its vendor and manufacture subdirectories e.g. `clueboard` is a directory for each keyboard project, for example `qmk_firmware/keyboards/clueboard/2x1800`. Within it, you'll find the following structure:
```

* `keymaps/`: Different keymaps that can be built
* `rules.mk`: The file that sets the default "make" options. Do not edit this file directly, instead use a keymap specific `rules.mk`.
* `config.h`: The file that sets the default compile time options. Do not edit this file directly, instead use a keymap specific `config.h`.

> * `info.json`: The file used for setting layout for qmk configurator see [Configurator Support](reference_configurator_support.md) for more information.

Reviewer: ```suggestion
* `info.json`: The file used for setting layout for QMK Configurator. See [Configurator Support](reference_configurator_support.md) for more information.
```

* `readme.md`: A brief overview of the keyboard.

> * `<keyboardName>.h`: This file is where the keyboard layout is defined against the keyboards physical matrix.

Reviewer: ```suggestion
* `<keyboardName>.h`: This file is where the keyboard layout is defined against the keyboard's switch matrix.
```

* `<keyboardName>.c`: This file is where you can find custom code for the keyboard.  

> For more information on project structure see [QMK Keyboard Guidelines](hardware_keyboard_guidelines.md).

Reviewer: ```suggestion
For more information on project structure, see [QMK Keyboard Guidelines](hardware_keyboard_guidelines.md).
```

Author: Pesky little comma's. Glad someone can grammer
